tools/vera++/scripts/rules/jerry_dereference_operator_always_on_right.tcl

```diff
@@ -18,7 +18,9 @@
 foreach f [getSourceFileNames] {
     set lineNumber 1
     foreach line [getAllLines $f] {
-        if {[regexp {\w\*\s\w+} $line]} {
+        if {[regexp {\w\*\s\w+} $line]
+             || [regexp {\w\*\)} $line]
+             || [regexp {\w\*$} $line]} {
```

Comment:
I got badly misled by the name of the rule. The first/old pattern matches something like `x* abc` (or, more realistically, the tail of `void* p`). The new ones match `void*)` and `void*`-at-end-of-line. If I'm not mistaken, none of these are pointer dereferences but pointer declarations.

As for the original pattern: why does `\s` have no `+` multiplier? And why does the second `\w` have one? Would changing these cause too many false positives?

If we are already touching this file, it would be good to get it right.